.github/workflows/ci.yml

```diff
@@ -170,12 +171,8 @@ jobs:
           
           # dont use golangci-lint
           # see: https://twitter.com/dominikhonnef/status/1394766501157167112
-          /home/runner/go/bin/golangci-lint run --no-config --enable=bodyclose,exhaustive,exportloopref,gochecknoglobals,tparallel,unparam ./...
-
-          mkdir -p /tmp/dgryski/semgrep-go
-          wget -nc --output-document=/tmp/dgryski/semgrep-go/rules.go https://raw.githubusercontent.com/dgryski/semgrep-go/master/ruleguard.rules.go
           go get github.com/quasilyte/go-ruleguard/dsl
-          /home/runner/go/bin/ruleguard -c 1 -rules /tmp/dgryski/semgrep-go/rules.go ./...
```

Comment:
changed because `go-ruleguard` does not have a way to ignore per line, see: https://github.com/quasilyte/go-ruleguard/issues/248     
So we have to use `golangci-lint`.

We need to ignore some parts in; https://github.com/komuw/goweb/blob/2c320d7d73f1b8336977ec07b8d744fc8862d84d/errors/errors_test.go#L53-L59